playground/backend/internal/code_processing/code_processing.go

```diff
@@ -76,13 +76,17 @@ func Process(ctx context.Context, cacheService cache.Cache, lc *fs_tool.LifeCycl
 		return
 	}
 
-	// compile
-	logger.Infof("%s: Compile() ...\n", pipelineId)
-	compileCmd := exec.Compile(ctxWithTimeout)
-	go processCmd(compileCmd, successChannel, errorChannel, dataChannel)
-
-	if err := processStep(ctxWithTimeout, pipelineId, cacheService, cancelChannel, successChannel, dataChannel, errorChannel, pb.Status_STATUS_COMPILE_ERROR, pb.Status_STATUS_EXECUTING); err != nil {
-		return
+	switch sdkEnv.ApacheBeamSdk {
+	case pb.Sdk_SDK_JAVA:
+	case pb.Sdk_SDK_GO:
+		// compile
```

Comment:
We don't need to compile java code?

Reply:
Agree, I think this way you miss Java SDK

Reply:
You can do it the following way:
```go
case pb.Sdk_SDK_JAVA, pb.Sdk_SDK_GO:
```

Reply:
yep, sorry, confused with python